test/cypress/components/paying-for-college.js

```diff
@@ -0,0 +1,37 @@
+export class PayingForCollege {
+
+  openLoanOption( name ) {
+    cy.visit( `/paying-for-college/choose-a-student-loan/#${ name }` );
+  }
+
+  openMoneyOption( name ) {
+    cy.visit( `/paying-for-college/manage-your-college-money/#${ name }` );
+  }
+
+  clickOption( name ) {
+    cy.get( 'span' ).contains( name ).click();
+  }
+
+  selectOption( name ) {
+    cy.get( '.bubble-top-text' ).contains( name ).click();
+  }
```

Comment:
Let's call these bubbles, to match the class nomenclature.

```suggestion
  clickBubble( name ) {
    cy.get( '.bubble-top-text' ).contains( name ).click();
  }
```

Reply:
Changed Option to Bubble